plugins/gateway/kong/README.md

```diff
@@ -1,5 +1,75 @@
 ## APIClarity Kong plugin
 
+### Prerequisite
+* APIClarity backend is running
+* Kong gateway is running in your K8s cluster, and configured correctly.
+
+If you just want to experience with it, and you don't have kong, you can take these few quick steps that will help you to quickly setup a running environment:
+1. Deploy sock-shop app:
+    ```shell
+       kubectl create namespace sock-shop
+       
+       kubectl apply -f https://raw.githubusercontent.com/microservices-demo/microservices-demo/master/deploy/kubernetes/complete-demo.yaml
+    ```
+2. Deploy Kong:
+    - Using helm3:
+    ```shell
+       helm install kong/kong --generate-name --set ingressController.installCRDs=false
+    ```
+    - Using kubectl:
+    ```shell
+       kubectl create namespace kong
+       kubectl apply -f https://bit.ly/kong-ingress-dbless
+    ```
+3. Wait for pod to be ready:
+    ```shell
+       watch kubectl get pods -n kong
+    ```
+4. Configure Ingress:
+    ```shell
+       kubectl apply -f - <<EOF
+       apiVersion: networking.k8s.io/v1
+       kind: Ingress
+       metadata:
+         name: catalogue
+         namespace: sock-shop
+       spec:
+         ingressClassName: kong
+         rules:
+         - http:
+             paths:
+             - path: /
+               pathType: Prefix
+               backend:
+                 service:
+                   name: catalogue
+                   port:
+                     number: 80
+       EOF
+    ```   
+5. Deploy APIClarity Kong Plugin:
```

Comment:
I think it should be more like https://github.com/apiclarity/apiclarity/tree/master/plugins/gateway/tyk#installation-using-a-pre-built-image to include also the helm installation

Reply:
Its mentions in later section. here it is just a quick installation